### PR TITLE
Refactored Back Office controllers

### PR DESCRIPTION
**Pull request by mickaelandrieu**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Removed deprecated calls to `$this->container` + used `$this->trans()` helper 
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| How to test?  | Nothing to do tests must pass as before.

> Why this is important? Because we are migrating all pages, and I see some deprecations copy/pasted everywhere :-)

#### Important guidelines

* Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
* Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
* Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/8924)

### CODE REVIEW

File: src/PrestaShopBundle/Controller/Admin/ModuleController.php [hunk 1]
```diff
@@ -782,15 +752,14 @@ private function getAddonsConnectToolbar()
                 'href' => '#',
                 'desc' => $addonsEmail['username_addons'],
                 'icon' => 'exit_to_app',
-                'help' => $translator->trans('Synchronized with Addons marketplace!', array(), 'Admin.Modules.Notification'),
-                'data-trololo' => 'lol',
```
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Controller/Admin/ModuleController.php:786]: @Quetzacoalt91 I've removed this entry, I hope this is not really used :trollface:

**Comment by alegout** [comment]
Reviewed 17 of 17 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://beta.reviewable.io/reviews/prestashop/prestashop/8924)*

**Review by alegout — APPROVED** [review]
Thank you @mickaelandrieu!

File: src/PrestaShopBundle/Controller/Admin/ModuleController.php (hunk 1 above)
Comment (Quetzacoalt91) [reply src/PrestaShopBundle/Controller/Admin/ModuleController.php:786]: NOOOOO

**Comment by mickaelandrieu** [comment]
Thanks @alegout !

**Pull request merged**